fms/tests/test_fms.py

```diff
@@ -6,7 +6,7 @@
 from happi.errors import SearchError
 from happi.item import OphydItem
 
-from fms.__main__ import add_fms_sensor, add_src_controller, delete_sensor
+from fms.__main__ import add_fms_sensor, delete_sensor
```

Comment:
Maybe in the future we move these functions outside of the __main__.py file, it's bad form to have to import from these double-underscore files.